__tests__/app.test.js

```diff
@@ -487,3 +483,43 @@ test("should handle not found when the article_id does not exist", (done) => {
     .send({ inc_votes: 2 })
     .expect(404, done)
 })
+
+// 9
+
+test("should delete a comment by comment_id", (done) => {
+  request(app)
+    .delete("/api/comments/1")
+    .expect(204)
+    .end((err, res) => {
+      if (err) return done(err)
+
+      expect(res.body).toEqual({})
+      done()
+    })
+})
+
+describe("deleteCommentById ()", () => {
+  test("should handle not found when the comment_id does not exist", (done) => {
+    request(app)
+      .delete("/api/comments/999")
+      .expect(404)
+      .end((err, res) => {
+        if (err) return done(err)
+
+        expect(res.body.msg).toBe("Not Found - Comment not found")
+        done()
+      })
+  })
+
+  test("should handle other errors when deleting a comment", (done) => {
+    request(app)
+      .delete("/api/comments/invalidCommentId")
+      .expect(400)
+      .end((err, res) => {
+        if (err) return done(err)
+
+        expect(res.body.msg).toBe("Bad Request")
+        done() // NC mentors - this is a call back provided by jest, it signals that the test has completed. it is used for handling async tasks i find it useful for me for readiabilty.
+      })
```

Comment:
Nice sad path tests :) but again, no need for the end block, you can just do a .then and expects inside. No need for done() either 👍